src/index.css

```
@@ -2,15 +2,174 @@
   --radius-lg: var(--radius);
   --radius-md: calc(var(--radius) - 2px);
   --radius-sm: calc(var(--radius) - 4px);
+
+  /* Custom breakpoints */
   --breakpoint-xs: 480px;
 }
 
+/* Light mode color values - default */
+@layer base {
+  :root {
+    --color-background: theme(--color-white);
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)